### ISSUE TITLE
bootloops with patched magisk boot.img but boots normally with normal boot.img

### ISSUE THREAD

**Issue by SuckyBuckyYT** [4accccc/vivo-Magisk-suu#16]
Device: vivo y91c or 1820
Android version: 8.1
Magisk version name: v27.0
Magisk version code: 27001

Title says all

**Comment by SuckyBuckyYT** [comment]
> [@SuckyBuckyYT](https://github.com/SuckyBuckyYT) hey, please teach me how to ubl then root my vivo y91c(1820)

Its impossible to do it on this specific vivo phone, they even made brom mode impossible to access for people without knowledge in this range, like before you could hold down v+ and v- in preloader and it would go into brom but after software update they patched that and Now the only way you can do so is by connecting this point shown in this website https://www.gsm1x.xyz/2022/10/testpoint-vivo-y91c-brom-new-sec.html to ground and THATS the only way to get into brom, But.... its all useless even if you get into brom and get the boot.img file there is No easily available tool to patch it, i dont think this phone uses suu because this project didnt work with it and it def doesnt use su so you dont even know what to do, and like there is basically No knowledge regarding these topics i talked with a few people way more experience than me about this 2 years ago and they tried to help me and even they didnt know how to do it and ended up ghosting me afterwards, its just a waste of time. the phone is old enough now, Root has become overrated also so better just give up. Btw sorry for the slow reply i dont often check my github

**Comment by deadcoder0904** [comment]
@SuckyBuckyYT did u ever find a solution to this?

i'm facing the same issue. i see orange state on my vivo & it restarts over & over again.

wasted so much time on this.

**Comment by SuckyBuckyYT** [comment]
> [@SuckyBuckyYT](https://github.com/SuckyBuckyYT) did u ever find a solution to this?
> 
> i'm facing the same issue. i see orange state on my vivo & it restarts over & over again.
> 
> wasted so much time on this.

No, I'll just mention you the important points from what I learned, the phone doesn't use suu or su it uses something else that's unknown that's why it's bootlooping, so you have to root according to what it uses which we don't know even if we did how are we gonna do that. So yeah just put in the old boot.img back your phone should recover

**Comment by deadcoder0904** [comment]
Ahh, sucks. Anyways, I'll just buy a phone that is easy to root now. No solution for this at all haha.

**Comment by FranklinAzriel** [comment]
Okay i got the same issue even after use https://github.com/4accccc/vivo-4.x-kernel-autopatch
maybe if someone wants check consoleramoops

[a.txt](https://github.com/user-attachments/files/26781515/a.txt)
